fe/fe-core/src/main/java/com/starrocks/qe/ConnectContext.java

```diff
@@ -1541,4 +1541,8 @@ public void onQueryFinished() {
             }
         }
     }
+
+    public boolean isArrowFlightSQL() {
```

Comment:
Using isChildName in the parent class is not a very scalable coding style, and I think it has no practical significance. You can abstract the specific method and override the parent class method, or use instanceof.